src/cycax/cycad/assembly_stl.py

```diff
@@ -0,0 +1,117 @@
+import json
+
+class AssemblyStl:
```

Comment:
The fact that OpenSCAD does the rendering is probably more important than that its STL. Thus consider a different name e.g. AssemblerOpenSCAD.